mysql.py

```diff
@@ -133,6 +133,7 @@
 	'Qcache_queries_in_cache': 'counter',
 	'Qcache_total_blocks': 'counter',
 	'Questions': 'counter',
+	'Read_only': 'gauge',
```

Comment:
Read_only is not a part of global status.
You can check by running 
show global variables like 'read_only'
show global status like 'read_only'